_configuration/configuration.md

```diff
@@ -444,12 +453,10 @@ For SCARA use the sample configuration in the `example_configurations/SCARA` fol
 // Enable this option for Toshiba steppers
 //#define CONFIG_STEPPERS_TOSHIBA
 ```
+Leave this option disabled for standard NEMA steppers.
```

Comment:
CONFIG_STEPPERS_TOSHIBA deals with stepper __drivers__. NEMA is a norm for stepper __motors__.
This sentence compares apples with pears.
The TOSHIBA stepper drivers need exceptional long times for the step-pulses.
We do limit the step frequency.